nototools/generate_dingbats_html.py

```diff
@@ -667,6 +679,7 @@ def context_string(codelist, cp):
         line.append('<td>&nbsp;')
     line.append('<td class="age">%s' % unicode_data.age(cp))
     name = _flagged_name(cp, flag_sets)
+    name = _break_name(name)
     line.append('<td class="name">%s' % name)
```

Comment:
FYI: You can use n-th-child selector to avoid adding ```class="<class name>"``` for every cell. 
See https://developer.mozilla.org/ko/docs/Web/CSS/:nth-child